webservices/tasks/refresh.py

```diff
@@ -19,7 +19,6 @@ def refresh():
             manage.update_functions()
```

Comment:
We need to remove this call since this method is no longer defined in `manage.py`; this is what's currently causing the nightly updates to fail in the `dev` environment. :-)